web/components/source-item.js

```diff
@@ -16,34 +16,33 @@ template.innerHTML = `
 #card {
 	display: grid;
 	grid-template-columns: 1fr 1fr;
-	row-gap: 5px;
+	row-gap: 10px;
 	position: relative;
 	box-sizing: border-box;
 	min-width: 5.14em;
 	min-height: 50px;
 	margin: 0.2em;
+	margin-top: 13px;
 	text-align: center;
 	user-select: none;
 	cursor: pointer;
 	padding: 0.7em 0.57em;
-	background-color: var(--background-color, white);
 	color: #333333;
 }
 
 #name {
 	font-size: 1.2em;
 	text-align: left;
+	min-height: 10px;
+	align-content: center;
```

Comment:
Since `font-size` is using a relative unit, consider using a relative unit for the `min-height` as well

Reply:
not needed... removing, thanks